azure/cloud/Keyvault.rego

```diff
@@ -34,7 +34,7 @@ azure_issue["KeyVault"] {
     resource := input.resources[_]
     lower(resource.type) == "microsoft.keyvault/vaults"
     count([c | r := input.resources[_];
-              r.type == "microsoft.keyvault/vaults/accessPolicies";
+              lower(r.type) == "microsoft.keyvault/vaults/accessPolicies";
```

Comment:
@rezoan `Policies` should be lower case as well

Reply:
fixed